github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -83,20 +83,17 @@ postsubmits:
                 memory: "1Gi"
               limits:
                 memory: "1Gi"
-    - name: publish-multiarch-bootstrap-image
+    - name: publish-bootstrap-image
       always_run: false
       run_if_changed: "images/golang/.*|images/bootstrap/.*"
       annotations:
         testgrid-create-test-group: "false"
       decorate: true
-      decoration_config:
-        grace_period: 5m0s
-        timeout: 5h0m0s
```

Comment:
dont we need timeout of lets say 1h still ?
(or whatever it takes + margin)

Reply:
as long as there wont be timeouts i am fine with it

Reply:
I believe there is a default timeout of 2 hours when it is not specified.

https://github.com/kubevirt/project-infra/blob/91d8af2824c646199697ebdd83933fc8c441b3d7/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L9